### ISSUE TITLE
Reading Progress Bar Component

### ISSUE THREAD

**Issue by theBeginner86** [ALPHAVIO/Mirage-UI#48]
### Feature request-
This progress bar should display the amount of content read by the reader and the amount which is left. This would further add motivation to the reader to finish what they are reading by scrolling down.

### Additional context

https://user-images.githubusercontent.com/73700530/121836048-524e1b80-ccf0-11eb-9d95-a640df6e6179.mov

LGM-SOC

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

**Comment by bhoskarpooja** [comment]
When am going to pull request after solving my 2nd issue and 1st one is unreviwed. Am facing this problem how can i create PR?
![Screenshot (509)](https://user-images.githubusercontent.com/72018117/121949562-38571c00-cd76-11eb-8c41-d20e9e6e1e5f.png)

**Comment by theBeginner86** [comment]
> When am going to pull request after solving my 2nd issue and 1st one is unreviwed. Am facing this problem how can i create PR?
> ![Screenshot (509)](https://user-images.githubusercontent.com/72018117/121949562-38571c00-cd76-11eb-8c41-d20e9e6e1e5f.png)

I feel that you are facing this issue because you are using the same branch for two different PRs.
I would suggest you to checkout to a new branch for each new feature (or PR) and delete it once that PR is merged successfully. By doing this, your local repo would remain clean. Moreover, it is advised to always work on a separate branch than the master branch. 
For next time just keep this thing in mind. 👍🏻

**Comment by bhoskarpooja** [comment]
Ok thank you.
